chpl/chpl-service/pom.xml

```diff
@@ -319,120 +319,122 @@
-                        <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductAccessibilityStandard.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductDownloadResponse.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductQmsStandard.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchDetails.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductTargetedUser.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/Contact.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CQMResultCertification.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CQMResultDetails.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/Developer.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/DeveloperStatus.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/DeveloperStatusEvent.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/Product.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/ProductOwner.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/ProductVersion.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/Surveillance.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceNonconformity.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceNonconformityDocument.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceNonconformityStatus.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceRequirement.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceRequirementType.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceResultType.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/SurveillanceType.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/TransparencyAttestationMap.java</source>
-                    </sources>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>schemagen</id>
-                        <goals>
-                            <goal>schemagen</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
+        <pluginManagement>
```

Comment:
What's `<pluginManagement>`?

Reply:
By wrapping the <plugins> tag with the <pluginsManagement> I was able to get rid pom.xml error that Eclipse has flagged in my project  since I started on the team.